modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -89,8 +90,7 @@ controlPlane:
 |Replicas sets the number of control plane (master) nodes included as part of the {product-title} cluster.
 
 ifeval::[{product-version} >= 4.4]
-a| [[provisioningNetworkInterface]]`provisioningNetworkInterface` |  | The name of the network interface on control plane nodes connected to the
-provisioning network.
+a| [[provisioningNetworkInterface]]`provisioningNetworkInterface` |  | The name of the network interface on nodes connected to the `provisioning` network. For {product-title} 4.9 and later releases, use `bootMACAddress` instead of `provisioningNetworkInterface`. Using `bootMACAddress` enables Ironic identify the IP address of the NIC and thereby communicate with the NIC without relying on a NIC name.
```

Comment:
Seems like there is a word missing in the last sentence. Should it be "enables Ironic _to_ identify the IP address"?